lib/bundle/bundle.rb

```diff
@@ -28,7 +28,7 @@ def mas_installed?
   end
 
   def mas_signedin?
-    Bundle.system "mas", "account"
+    Kernel.system "mas account &>/dev/null"
```

Comment:
I'm OK with this but a feel a little weird about it. We've defined our own `system()` just above, should this instead use that and that method have a default parameter that shortcuts like `if ARGV.verbose?`

Reply:
@colindean I tried to refactor accordingly but given the behaviour we want here (don't ever print any output regardless) it felt better to just use this. If there was another 1-2 uses in this form I'd be 👍 on adding a separately method.